packages/web/package.json

```diff
@@ -44,10 +44,12 @@
     "react-places-autocomplete": "^7.3.0",
     "react-rating": "^2.0.5",
     "react-select": "^3.1.0",
+    "react-slick": "^0.27.11",
     "react-spinners": "^0.9.0",
     "react-tabs": "^3.1.1",
     "react-toastify": "^6.0.8",
     "react-tooltip": "^4.2.6",
+    "slick-carousel": "^1.8.1",
```

Comment:
Esse pacote é necessário mesmo? Estamos incluindo só o CSS?

Reply:
@nicholasio sim, é só pra incluir o CSS. Qualquer coisa posso pegar o CSS dele e colocar na pasta de styles.

Reply:
Se não for muita coisa, acho interessante fazer isso, evita de quebrar nossos styles caso a lib atualiza e evite mais uma dependência.